src/lld/moviebooking/class_diagram.puml

```diff
@@ -1,3 +1,323 @@
 @startuml
+'Main Entity and Relational mapping entity
 
+package com.movie_booking.entity.main{
+    class Identifier{
```

Comment:
Identifier is too abstract? what it helps to identifies? Does Identifier helps User, Crew, Movie with common functionality, in that case move it to abstract class? or if not move it to interface? 
Having a class which by itself cannot be used directly without extends should be either abstract or interface

Reply:
moved it to abstract, as id and name field and its related operation can be re-used across user, crew, movie, and other models which required id and name and common operation on the fields.